006 to do bzw. Einkaufsliste | I.B. E.G.

Lucerrion opened the pull request:
006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
[006] To-Do bzw. Einkaufsliste files added |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

Lucerrion committed:
[006] nav id added |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
[006] nav ul id added |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
Styleguide mistake fixed

Lucerrion committed:
[006] Johnnys JS file added and combined |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
[006] Input warning styled |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
[006] textarea instead of input |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
[006] innerHTML changed to innerText |

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

Lucerrion committed:
[006] error message position changed |  I.B  E.G.

006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f

SpaTeamAcademy commented:
Es ist noch möglich Leerzeichen (oder leeres Enter) zu der Taskliste hinzuzufügen.

SpaTeamAcademy commented:
Bitte merge conflict lösen.

SpaTeamAcademy commented:
> Es ist noch möglich Leerzeichen (oder leeres Enter) zu der Taskliste hinzuzufügen.

Nicht Teil der Story. OK.
 
**! Bitte Merge Conflict lösen.**

Vanillekipferl committed:
[006] added delete button after new list entry

SpaTeamAcademy reviewed (commented):
**Achtung!**
Es wird gerade der Branch 006 bearbeitet, der noch nicht ins main gemerged wurde,

SpaTeamAcademy commented on to-do.js:
```diff
@@ -0,0 +1,26 @@
+        newItem.innerText = newTask;
+        document.getElementById("toDo-ul").appendChild(newItem);
+        var deleteButton = document.createElement("button");
+        deleteButton.innerText = "X";
```
Nicht Teil der Story 006!

Lucerrion committed:
Merge branch 'main' of github.com:SpaTeamAcademy/Team_A

# Conflicts:
#	index.html
#	rock-paper-scissor.html

Lucerrion committed:
Merge branch '006-To-Do-bzw.-Einkaufsliste' of github.com:SpaTeamAcademy/Team_A

Lucerrion committed:
[006] extra code deleted |
006 To-Do bzw. Einkaufsliste https://app.asana.com/0/1205072348884926/1205082382297347/f
